api/types.go

```diff
@@ -272,6 +285,8 @@ type Options struct {
 	PresencePenalty  float32  `json:"presence_penalty,omitempty"`
 	FrequencyPenalty float32  `json:"frequency_penalty,omitempty"`
 	Stop             []string `json:"stop,omitempty"`
+
+	Think bool `json:"think,omitempty"`
```

Comment:
Did you mean to add this to `Options`?

Reply:
oops this looks like a remnant from an earlier version. Will remove!